src/venv_manager.rs

```diff
@@ -43,28 +45,41 @@ const APP_INFO: AppInfo = AppInfo {
 
 impl VenvManager {
     #[allow(clippy::new_ret_no_self)]
-    pub fn new(project_path: std::path::PathBuf, python_info: PythonInfo) -> Result<Self, Error> {
+    pub fn new(
+        project_path: std::path::PathBuf,
+        python_info: PythonInfo,
+        settings: Settings,
+    ) -> Result<Self, Error> {
         let lock_path = project_path.join(LOCK_FILE_NAME);
         let setup_py_path = project_path.join("setup.py");
-        let venv_path = Self::get_venv_path(&project_path, &python_info.version)?;
+        let venv_path = Self::get_venv_path(
+            &project_path,
+            &python_info.version,
+            settings.venv_outside_project,
+        )?;
         let paths = Paths {
             project: project_path,
             venv: venv_path,
             lock: lock_path,
             setup_py: setup_py_path,
         };
-        let venv_manager = VenvManager { paths, python_info };
+        let venv_manager = VenvManager {
+            paths,
+            python_info,
+            settings,
+        };
         Ok(venv_manager)
```

Comment:
Hmm I might just `Ok(VenvManager { })` because I like to save vertical space real estate, but not a big deal

Reply:
Actually, new() no longer needs to return a Result :)

Reply:
Awesome!